Lib/defconQt/featureTextEditor.py

```diff
@@ -17,11 +17,11 @@ def __init__(self, font=None, parent=None):
         self.editor = FeatureTextEditor(self.font.features.text, self)
         self.resize(600, 500)
 
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&Save…", self.save, QKeySequence.Save)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&Save") + "…", self.save, QKeySequence.Save)
```

Comment:
It is a bad l10n practice in general to split translatable strings, you don’t know what languages will want to do with the string (reorder it, add spaces, use different punctuation, and so on).